recipes-kernel/jool/jool_git.bb

```diff
@@ -7,7 +7,7 @@ LIC_FILES_CHKSUM = "file://${S}/COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 
 SRC_URI = "git://github.com/NICMx/Jool.git"
 
-SRCREV = "89b3c1098d2e05a67931805c6eed3182b03b8a83"
+SRCREV = "de72f5f5a15a7218dc7cd566b108f8f2323b3a3a"
```

Comment:
Since we're using a released base hash, mind also setting PV for the package version?

Something like:

`PV = "4.0.0"`

Reply:
Will do.